packages/openid4vc/src/openid4vc-issuer/router/credentialEndpoint.ts

```diff
@@ -274,7 +274,7 @@ export function configureCredentialEndpoint(router: Router, config: OpenId4VcIss
         },
         openId4VciVersion:
           issuerMetadata.originalDraftVersion === Openid4vciDraftVersion.Draft15 ||
-          issuerMetadata.originalDraftVersion === Openid4vciDraftVersion.Draft16
+          issuerMetadata.originalDraftVersion === Openid4vciDraftVersion.V1
             ? 'v1.draft15'
```

Comment:
shouldn't we update the openid4vci version here to v1? it's odd that v1 results in v1.draft15 being stored on the record?